src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -541,16 +693,16 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         });
     }
 
-    private async updateProjectTagsFromAsset(asset: IAssetMetadata) {
+    private async updateProjectTagsFromAsset(asset: IAssetMetadata, project: IProject, forceSave: boolean) {
```

Comment:
Why are we passing in a project vs. using the current project?
In what scenario do we need to force save? Is it to save predicted regions?  If so - we should refactor to make this fit into the existing data flow.

Reply:
The call with force save is commented and it's the thing I need to show you on debug.